mz_zip.c

```diff
@@ -2105,7 +2105,7 @@ int32_t mz_zip_entry_read_close(void *handle, uint32_t *crc32, int64_t *compress
     }
 
     /* If entire entry was not read verification will fail */
-    if ((err == MZ_OK) && (total_in > 0) && (!zip->entry_raw)) {
+    if ((err == MZ_OK) && (total_in == zip->file_info.compressed_size ) && (!zip->entry_raw)) {
```

Comment:
Has an extra space here at the end. After that i can merge.

Reply:
> Has an extra space here at the end. After that i can merge.

@nmoinvaz small question: have you thought about adding support for code formatting tool? For example, clang-format?